src/spetlr/db_auto.py

```diff
@@ -43,4 +45,6 @@ def getDbApi() -> WorkspaceClient:
     if not host or not token:
         raise Exception("Unable to auto-configure api client.")
 
-    return WorkspaceClient(host=host, token=token)
+    return WorkspaceClient(
+        host=host, token=token, product=spetlrname, product_version=spetlrversion
```

Comment:
How does product name look like?.. alphanum and dasges are fine. and I hope the version is SemVer-compatible. If so - lgtm

Reply:
@nfx thanks for your feedback.
productname is alphanum -> it is actually just "spetlr". I just didnt want to hardcode the values - rather import the library name.
The version is SemVer compatible.